testing/sdk_cmd.py

```diff
@@ -176,22 +184,26 @@ def svc_cli(
     return rc, stdout, stderr
 
 
-def _get_json_output(cmd, print_output=True, check=False):
-    _, stdout, stderr = run_cli(cmd, print_output, check=check)
+# def _get_json_output(cmd: str, print_output: bool = True, check: bool = False) -> Any:
```

Comment:
Why not remove this instead of commenting-out?

Reply:
Done